src/gui/mzroll/mainwindow.cpp

```diff
@@ -352,8 +352,10 @@ using namespace mzUtils;
 	logWidget = new LogWidget(this, std::cout);
 	// rconsoleDockWidget = new RconsoleWidget(this);
 	spectralHitsDockWidget = new SpectralHitsDockWidget(this, "Spectral Hits");
-    peptideFragmentation = new PeptideFragmentationWidget(this);
-
+	peptideFragmentation = new PeptideFragmentationWidget(this);
+	alignmentErrorDialog=new QErrorMessage(this);
```

Comment:
Let's not create QErrorMessage unnecessarily. Only create it when we actually want to show the error to the user

Reply:
@rish9511 made this as local variable